static/sass/_patterns_sticky_footer.scss

```diff
@@ -5,6 +5,7 @@
 
   html,
   body {
+    background: #fff;
```

Comment:
That's not a good place for this. Backround color is not related to sticky footer pattern. If at some point we remove sticky footer, backround would be removed as well.

Best to put it somewhere in main `styles.scss` with a comment that we wait until it's added to Vanilla as well (via https://github.com/vanilla-framework/vanilla-framework/issues/2129)